example/heroic.example.yml

```diff
@@ -1,287 +1,48 @@
-# vim: filetype=yaml
 port: 8080
 
 # Cluster configuration.
 cluster:
-  # Local ID.
-  # @default Generated UUID.
-  #id: 88b3f090-49f8-43fe-b30e-5ce0e3889884
-  # When communicating with self, avoid using the network.
-  # @default false
-  #useLocal: false
```

Comment:
is this a non-exist option?

Reply:
`useLocal`? That's an option, I documented it in the cluster section.

Reply:
I'm not seeing `useLocal` in any of the files other than this one?

Reply:
Line 215 of config.md. Are you not expanding that file? Its big and collapsed by default, but its the most important file of this PR.

Reply:
I see it now